testscript/runserver.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
```

Comment:
```suggestion
set -eu
```
Exit on error `-e`; treat usage of undefined variable as error `-u` (instead of empty string)